front/app/containers/Admin/sideBar/NotificationsPopup.tsx

```diff
@@ -93,18 +93,18 @@ export const NotificationsPopup = () => {
                   fontSize="base"
                   textAlign="left"
                   my="0px"
-                  w="100%"
                 >
                   {formatMessage({ ...messages.notifications })}
                 </Text>
               </>
             )}
-            <Box
-              w="auto"
-              h={isSmallerThanPhone ? '0px' : '18px'}
-              ref={iconDivRef}
-            >
-              {unreadNotificationsCount > 0 && (
+            {!isSmallerThanPhone && unreadNotificationsCount > 0 && (
+              <Box
+                w="auto"
+                h={isSmallerThanPhone ? '0px' : '18px'}
```

Comment:
Since we have the condition on line 101
```suggestion
                h="18px"
```